mu-plugins/central-hub/tests/phpunit/unit/config-store/strStartsWith.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ *  Tests for str_starts_with
+ *
+ * @package    spiralWebDb\centralHub\Tests\Unit\ConfigStore
+ * @since      1.3.0
+ * @author     Robert A. Gadon
+ * @link       https://github.com/rgadon107/cornerstone
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Unit\ConfigStore;
+
+use function KnowTheCode\ConfigStore\str_starts_with;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_StringStartsWith
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\ConfigStore
+ * @group   config-store
+ */
+class Tests_StringStartsWith extends Test_Case {
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/config-store/internals.php';
+	}
+
+	/*
+	 *  Test that a string starts with a character or substring.
+	 */
+	public function test_that_string_starts_with_a_character_or_substring() {
+		$needle   = 'Hello';
+		$haystack = 'Hello_world!';
+		$actual   = str_starts_with( $haystack, $needle, $encoding = 'UTF-8' );
+
+		$this->assertTrue( $actual );
+		$this->assertSame( true, $actual );
+	}
+}
```

Comment:
This test suite is missing tests for when the character or substring is not at the start of the string.  Remember to test both true and false conditions.  You want to capture all edge cases to ensure your code works as designed:

```php
/**
 * Test str_starts_with() should return `false` when string does not start with the given character or substring.
 */
public function test_should_return_false_when_does_not_start() {
    $this->assertFalse( str_starts_with( 'world!', 'Hello world!' );
    $this->assertFalse( str_starts_with( 'Hey', 'Hello world!' );
    $this->assertFalse( str_starts_with( 'Press', 'WordPress rocks!' );
   $this->assertFalse( str_starts_with( 'stone', 'Cornerstone' );
}
```